src/Explorer/Tabs/NotebookTabBase.ts

```diff
@@ -45,6 +50,20 @@ export default class NotebookTabBase extends TabsBase {
         contentProvider: this.container.notebookManager?.notebookContentProvider,
       });
     }
+    if (
+      options.notebookContentItem &&
+      NotebookUtil.getContentProviderType(options.notebookContentItem.path) ===
+        NotebookContentProviderType.JupyterContentProviderType
+    ) {
+      // There is no way to turn off auto-save, set to 1 year
+      NotebookTabBase.clientManager
+        ?.getStore()
+        .dispatch(configOption("autoSaveInterval").action(365 * 24 * 3600 * 1000));
```

Comment:
Are we using different stores for every tab?

Reply:
autoSaveInternal is a setting on store, and looks like we share the store so this setting is global for all the tabs. Can you please test if it works locally by setting the interval to like 5s?

Reply:
Removed this in this PR